ob-pattern-2/requirements.yaml

```diff
@@ -14,8 +14,8 @@
 
 dependencies:
   - name: mysql-ob
-    version: "1.5.0-1"
-    repository: "https://helm.wso2.com"
+    version: "2.0.0-1"
+    repository: "file://../database/mysql-ob"
```

Comment:
@nalakagunathilaka let's revert this change as we expect the pull the chart from the official WSO2 Helm chart repository, once uploaded.

Reply:
It was decided to keep local repository paths like that. @Aaquiff please comment on this.

Reply:
+1. @Aaquiff I suppose then we may have to follow the same approach in other product charts as well.